packages/search-engine/.env

```diff
@@ -0,0 +1 @@
+CONNECT_DB = 'mongodb+srv://jackodev:SashaDev2023$@first-step.z7r13.mongodb.net/searchEngineDB?retryWrites=true&w=majority'
```

Comment:
@JackoDev let's leave this empty because here we are leaking the username and password of the mongo instance that you have configured.

Reply:
Checked, sorry, my bad.